.github/ISSUE_TEMPLATE/lifecycle_template.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Specification Review Issue
+about: Start a request for a specification creation/plan/release... lifecycle review
+title: ''
+labels: creation,plan,progress,release
+assignees: ''
+---
+
+## Specification issue template
+When creating a specification project lifecycle review, create issues in the  with the content defined as follows.
+
+- [ ] Specification name and version
+- [ ] Add a label from one of: creation, plan, progress, release
```

Comment:
I am not sure why we need 4 labels. Besides, this labels are only suitable for spec releases. I think it is better to make it more specific by prefixing `SR-Creation` etc, so that other github issues won't use these labels. I think we only need three labels: `SR-Creation`, `SR-Progress`, `SR-Release`. When we first create the issue, label it `SR-Creation`. When making progress, change the label to `SR-Progress`. When it is ready to be released, change the label to `SR-Release`